File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    w=[]
    h=[]
    
    #가로, 세로 길이 중 긴 것을 w에 넣고 짧은 것을 h에 넣기
    for x in sizes:
        if x[0]>=x[1]:
            w.append(x[0])
            h.append(x[1])
        else:
            w.append(x[1])
            h.append(x[0])
            
    #두 개의 리스트에서 가장 큰 값끼리 곲하기
    answer=max(w)*max(h)
     
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        
        int small = 0;
        int big = 0;
        
        for(int i=0; i<sizes.length;i++){
            int a = sizes[i][0];
            int b = sizes[i][1];
            
            small = Math.max(small, Math.min(a,b));
            big = Math.max(big, Math.max(a,b));
        }
        
        answer = small * big;
        
        return answer;
    }
}